src/parser.rs

```diff
@@ -851,6 +852,95 @@ pub fn parse_jacoco_xml_report<T: Read>(
     Ok(results)
 }
 
+pub fn parse_gocov<T: Read>(
+    reader: &mut BufReader<T>,
+) -> Result<Vec<(String, CovResult)>, ParserError> {
+    let re = Regex::new(r"([^:]+):(\d+)\.\d+,(\d+)\.\d+ (\d+) (\d+)").unwrap();
```

Comment:
nit: we could compile this only once by saving it in a static variable (can be a follow-up)